src/consensus/params.h

```diff
@@ -15,6 +15,7 @@ namespace Consensus {
 enum DeploymentPos
 {
     DEPLOYMENT_TESTDUMMY,
```

Comment:
nit: probably to late now and kinda-unrelated to this PR, but instead of `TESTDUMMY` we could have used something like `DEFAULT` or `DEFAULTVOID`.